src/pngwriter.cc

```diff
@@ -2023,6 +2023,11 @@ void pngwriter::write_png(void)
 ///////////////////////////////////////////
 void pngwriter::plot_text( char * face_path, int fontsize, int x_start, int y_start, double angle, char * text, double red, double green, double blue)
```

Comment:
ah I see you try to keep the API the same in the other PR. Well, then it still adds to the API so I leave the label in for the change log.

still, I am not a big fan of adding more c-style string APIs. but this can be refactored in an independent PR.